### DISCUSSION TITLE
Arabic Language support

### DISCUSSION THREAD

**Discussion by Oscar404M** [GitbookIO/community#412] [Feature Requests]
Can you please add arabic langauge to your platform
with the right direction and language support !

**Comment by MadeByPP** [comment]
After contacting gitbook support, they responded that they currently have no plans to add support for other languages.

**Comment by aikoii** [comment]
👋 @Oscar404M Thanks for sharing your feedback.

We have a similar suggestion here #76, I encourage you to vote for it. That said, unfortunately, we don't have any immediate plans to add a new language, but we may offer a better RTL support in future 🤞.

(will be closing this in favor of the first request)
